### ISSUE TITLE
Another imageio.ffmpeg.download error

### ISSUE THREAD

**Issue by MatthewYounkins** [Zulko/moviepy#960]
Asked at StackOverflow but I guess this is a fairly specific question that they can't help with?

https://stackoverflow.com/questions/56178559/imageio-ffmpeg-download-requires-a-different-install-of-imageio-and-imageio-w

### Expected Behavior

I'd like to run VideoFileClip from moviepy.  I'm running Spyder through Anaconda.  I attempt to run it, but I get the error below....

### Actual Behavior

>RuntimeError: imageio.ffmpeg.download() has been deprecated. Use 'pip install imageio-ffmpeg' instead.'

When I try `pip install imageio-ffmpeg`, I get:

>Requirement already satisfied: imageio-ffmpeg in c:\programdata\anaconda3\lib\site-packages (0.3.0)

Other answers recommend `sudo pip install imageio==2.4.1`. I get the error:

```
moviepy 1.0.0 has requirement imageio<3.0,>=2.5; python_version >= "3.4", but you'll have imageio 2.4.1 which is incompatible.
Installing collected packages: imageio
  Found existing installation: imageio 2.5.0
Cannot uninstall 'imageio'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```

If I try `conda remove imageio` I eventually get...

```
The following packages will be REMOVED:

  imageio-2.5.0-py37_0

  moviepy-0.2.3.5-py_0

  scikit-image-0.14.2-py37ha925a31_0
```
So I'm concerned that if I actually remove imageio and install v2.4.1, I'll auto-remove moviepy, and then when I eventually reinstall moviepy, it will bump my imageio up to 2.5 again, completing the circle of incompatibility.

### Steps to Reproduce the Problem
I believe you'd need to be running Anaconda to get the 'problem' which is really that Conda thinks it needs 2.5.0 but can't actually work with 2.5.0.

Thoughts?

**Comment by vic1309** [comment]
Did you solve the problem?

**Comment by arianaa30** [comment]
I have the same problem.

**Comment by jayrodge** [comment]
were you able to solve this problem?

**Comment by m-a-robinson** [comment]
I have the same problem.

**Comment by drugescu** [comment]
This fixed the problem for me: 

`conda install -c conda-forge imageio==2.4.1`

**Comment by fancydancing** [comment]
Hi, I've also faced this problem. I had `moviepy 0.2.3.5` installed along with `imageio 2.4.1`. Everything was fine but then I started to get errors like here:
[(https://github.com/Zulko/moviepy/issues/1078)]
saying that I should update the `moviepy` package. These errors happen when I'm trying to use `save_frame` from `VideoFileClip`.

So I updated `moviepy`, it forced update of `imageio` and now I'm here. Is there any way to get rid of both errors at the same time?)
